### PR TITLE
Homepage of COVIDWatch website

### PR DESCRIPTION
**Pull request by michaelolubajo**
I created homepage, added images and introductory text of news articles.
Screenshot_2020-08-01 COVIDWatch Homepage

### CODE REVIEW

**Review by yemexx1 — COMMENTED** [review]
Looks great but please get an approval from a teammate.

**Review by nwakaemecynthia — APPROVED** [review]

File: covidWatch/home.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>COVIDWatch Homepage</title>
+</head>
+<body>
+	<a href="#"><img src="images/logo.png"></a>
```
Comment (yemexx1) [inline covidWatch/home.html]: Can we add an image alt here? Just incase the image doesn't load.

File: covidWatch/home.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  		<a href="#">HOME</a>
```
Comment (yemexx1) [inline covidWatch/home.html]: Let's align this. You can use something like https://www.freeformatter.com/html-formatter.html to get a good formatting. Ideally your IDE should also have a formatting option if you already use one.

File: covidWatch/home.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  	<h2><a href="#">NIGERIA, IMF AND COVID-19</a></h2>
+  	<img src="images/news4.jpg">
+  	<p>
+  		<b>Source:</b> CISLAC Nigeria <br>
+  		<b>Date:</b> 16th July, 2020 <br>
+  		How Nigeria could avoid waste and corruption when spending billions in IMF emergency assistance during COVID-19
+		Amidst a catastrophic collapse of oil prices and a health care system unable to cope with the COVID-19 pandemic, Nigeria recently requested US$3.4 billion in emergency assistance from the International Monetary Fund (IMF). This massive loan, which was approved in April, adds to billions of dollars in emergency assistance and should avert catastrophe for Africa's largest economy and population of over 200 million.
+		<a href="#">Read more</a>
+  	</p>
+
```
Comment (yemexx1) [inline covidWatch/home.html]: We can remove these empty lines

**Review by yemexx1 — CHANGES_REQUESTED** [review]
Looking good in general @michaelolubajo. I will suggest we move the files to the root of the project, no need to have it in a folder for now. Images can of course stay in a folder.

File: covidWatch/home.html (hunk 2 above)
Comment (yemexx1) [inline covidWatch/home.html]: You could use a nav here for the links: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_nav

Comment (michaelolubajo) [reply covidWatch/home.html]: Done

**Review by nwakaemecynthia — APPROVED** [review]

**Comment by nwakaemecynthia** [comment]
I noticed you have affected the changes as requested by Yemi.
However our `nav` are not uniform.
@elizabethbusola , please help align this when linking our documents.
if you need help on that hit me up on slack.

Thank you team.

**Review by yemexx1 — APPROVED** [review]

**Pull request merged**